contributing/development/core_and_modules/custom_audiostreams.rst

```diff
@@ -49,44 +49,55 @@ Therefore, playback state must be self-contained in AudioStreamPlayback.
 
 	/* audiostream_mytone.h */
 
-	#include "core/reference.h"
-	#include "core/resource.h"
-	#include "servers/audio/audio_stream.h"
+	#ifndef AUDIOSTREAMMYTONE_H
+	#define AUDIOSTREAMMYTONE_H
+
+	#include <godot_cpp/classes/audio_stream.hpp>
+	#include <godot_cpp/godot.hpp>
+
+	namespace godot {
 
 	class AudioStreamMyTone : public AudioStream {
 		GDCLASS(AudioStreamMyTone, AudioStream)
 
 	private:
 		friend class AudioStreamPlaybackMyTone;
-		uint64_t pos;
+		float pos;
 		int mix_rate;
 		bool stereo;
 		int hz;
```

Comment:
It may be worth assigning default values here instead of the constructor to avoid issues with undefined behavior if one of those is accidentally unset.